components/brave_wallet_ui/page/screens/composer_ui/virtualized_token_list/virtualized_tokens_list.tsx

```diff
@@ -18,30 +26,35 @@ import {
 // Styled Components
 import {
   AutoSizerStyle,
-  ListItemWrapper,
   listItemInitialHeight
 } from './virtualized_tokens_list.style'
 
 interface VirtualizedTokensListProps {
-  tokenList: BraveWallet.BlockchainToken[]
+  tokenList: UserAssetInfoType[]
   selectedFromToken?: BraveWallet.BlockchainToken
   selectedToToken?: BraveWallet.BlockchainToken
+  selectingFromOrTo: 'from' | 'to'
+  spotPriceRegistry?: SpotPriceRegistry
+  isLoadingSpotPrices?: boolean
```

Comment:
any reason why we want this as a prop instead of running the query from in the list?

Reply:
We only need to query `spotPrices` for tokens that have a `balance`, is already handled by the `parent` component and would create duplicate logic if we did so.